docs/content/docs/grype/_index.md

```diff
@@ -1,49 +1,47 @@
 ---
-title: "Beta - Grype Integration"
+title: "Grype Integration"
 linkTitle: "Grype Vulnerability Scanner"
 weight: 1
 ---
 
-Anchore Engine 0.10 includes a limited-functionality beta release of an integration with [Grype](https://github.com/anchore/grype)
```

Comment:
Nit: "replaces the legacy vulnerability scanner in previous versions" -> "replaces the legacy vulnerability scanner used in previous versions". The legacy scanner will still be used for anyone running an older version of engine.

Reply:
I made this change, Thank you.